implements cli options for node count settings

almartin82 opened the pull request:
There has been a fair amount of conversation in the keybase chat about reducing the number of peers as a way to resolve full nodes that are getting slammed and knocked out of sync.  This draft PR attempts to streamline that process by creating command line options for update the `target_peer_count` and `target_outbound_peer_count` settings in `config.yaml`.

Some questions I have for my reviewer(s)!

1. Is this the desired way to change node counts for performance/sync issues?  I see `inbound_rate_limit_percent` as a top-level key in the yaml - would that be a preferable strategy for turning down the inbound traffic to a node?

2. Is there anywhere where I can read about the project's philosophy about which config options get CLI arguments and which ones should be handled via edits to the yaml directly?

Will keep this draft until I add tests.

almartin82 committed:
implements cli options for node count settings

almartin82 commented:
@nirajpathak13 I initially opened this change on #3811, which was based off of `1.1.4`, but there were meaningful changes to `configure.py` (adding `testnet`) so I closed that and re-opened this fresh off of main.  You were tagged on the original PR so thought I would mention you here 🙇.

almartin82 committed:
resolve flake8 error in what appears to be the house style for line-too-long function defs

mariano54 commented:
1. No, the rate limit parameters should not be changed.
2. I think we need to rewrite configure to allow changing any field in the yaml. like `configure full_node.target_peer_count=60` or something like that. Adding one a time is a lot of work and error prone. 
Anyway I don't see any harm in adding one more for now. @justinengland thoughts?

wjblanke commented:
I think this could go in, except for the lint issue

https://github.com/Chia-Network/chia-blockchain/pull/3943/checks?check_run_id=2518593661#step:4:153

hoffmang9 committed:
Create configure.py

`pip install black`
or online https://black.readthedocs.io/en/stable/getting_started.html#try-it-out-online

hoffmang9 commented:
I applied black to this. `pip install black` or https://black.readthedocs.io/en/stable/getting_started.html#try-it-out-online

hoffmang9 committed:
Actually black within the repo configuration for black

hoffmang9 reviewed (dismissed):
AOK

hoffmang9 committed:
Modify defaults to 8 and 80

hoffmang9 reviewed (approved):
Re approving after @wjblanke change request
